android/src/main/java/com/mrousavy/camera/CameraView.kt

```diff
@@ -410,16 +412,21 @@ class CameraView(context: Context, private val frameProcessorThread: ExecutorSer
         // User has selected a custom format={}. Use that
         val format = DeviceFormat(format!!)
         Log.i(TAG, "Using custom format - photo: ${format.photoSize}, video: ${format.videoSize} @ $fps FPS")
-        previewBuilder.setTargetResolution(format.videoSize)
+        if (video == true) {
+          previewBuilder.setTargetResolution(format.videoSize)
+        } else {
+          previewBuilder.setTargetResolution(format.photoSize)
+        }
         imageCaptureBuilder.setTargetResolution(format.photoSize)
-        imageAnalysisBuilder.setTargetResolution(format.videoSize)
+        imageAnalysisBuilder.setTargetResolution(format.photoSize)
```

Comment:
ImageAnalysis uses images as input, not video, so it should be set to the resolution that image capture will be using.  This wasn't an issue before because `videoSize` and `photoSize` were equal to one another.